src/plugins/blocksmith/components/divider-block-modal/CustomOption.jsx

```diff
@@ -0,0 +1,32 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import './styles.scss';
+
+const CustomOption = ({ innerProps, data, selectProps, isSelected }) => (
+  <div {...innerProps} className="divider__color divider__option">
+    <div className={classNames('select__option', { 'is-selected': isSelected })}>
+      <div className="divider__option-label">{data.label}</div>
+      <div
+        className={`${data.value} ${selectProps.strokeColor || data.value} divider__bar`}
```

Comment:
@cohencodes do you need the first `data.value` or can you get by with just the `selectProps.strokeColor || data.value`?